cfgov/unprocessed/css/organisms/video-player.less

```diff
@@ -94,15 +96,24 @@
     }
 
     & > .cf-icon-svg {
-        // Ensuring play icon optically appears horizontally centered.
-        position: relative;
-        left: 0.09375em;
-
-        // Ensuring play icon is vertically centered.
-        vertical-align: bottom;
+        // Size and position visually centers the play arrow.
+        position: absolute;
+        left: 55%;
+        top: 50%;
+        transform: translate( -50%, -50% );
+        font-size: 30px;
```

Comment:
This is already inherited from the button element itself and isn't needed.
```suggestion
```

Reply:
Fixed